### ISSUE TITLE
Production webpack doesn't define necessary loaders

### ISSUE THREAD

**Issue by atomanyih** [pivotal-cf/pui-react-tools#24]
So to add loaders to webpack, we have to do this:

```js
const {module: {loaders: baseLoaders = []} = {}} = require('pui-react-tools/webpack/base');
const {module: {loaders: testLoaders = []} = {}} = require('pui-react-tools/webpack/test');
const {module: {loaders: devLoaders = []} = {}} = require('pui-react-tools/webpack/development');
const {module: {loaders: prodLoaders = []} = {}} = require('pui-react-tools/webpack/production');

module.exports = {
  webpack: {
    base: {
      module: {
        loaders: [
          {test: /\.json$/, loader: 'json'},
          ...baseLoaders
        ]
      }
    },
    test: {
      module: {
        loaders: [
          {test: /\.json$/, loader: 'json'},
          ...testLoaders
        ]
      }
    },
    development: {
      module: {
        loaders: [
          {test: /\.json$/, loader: 'json'},
          ...devLoaders
        ]
      }
    },
    production: {
      module: {
        loaders: [
          {test: /\.json$/, loader: 'json'},
          ...prodLoaders
        ]
      }
    }
  }
};
```

However, this breaks in production because the production webpack config doesn't define the loaders it needs. Unlike other environments, `...prodLoaders` does not ensure that you have all the loaders PUI provides by default.

Instead, we have to include the base loaders to prod manually:

```js
// ...other requires
const {module: {loaders: prodLoaders = baseLoaders} = {}} = require('pui-react-tools/webpack/production');

module.exports = {
  webpack: {
    // ...other envs
    production: {
      module: {
        loaders: [
          {test: /\.json$/, loader: 'json'},
          ...prodLoaders
        ]
      }
    }
  }
};
```

This is kind of unfortunate because it's inconsistent with other environments. If `pui-react-tools/webpack/production.js` included the base loaders it would at least be consistent. 

similar to https://github.com/pivotal-cf/pui-react-tools/issues/22

**Comment by charleshansen** [comment]
I see, yeah, we can add this. The implementation is a little silly, but I see how it helps. Ideally if we fix #22 this won't matter.

**Comment by charleshansen** [comment]
You control production webpack yourself now in version 3 (we fixed #22)
